crates/bifrost/src/loglet/mod.rs

```diff
@@ -154,6 +154,22 @@ pub trait LogletReadStream: Stream<Item = Result<LogEntry<LogletOffset>, Operati
 
 pub type SendableLogletReadStream = Pin<Box<dyn LogletReadStream + Send>>;
 
+#[allow(dead_code)]
+pub(crate) struct Resolver {
```

Comment:
To make it obvious that this a dual of LogletCommit, can we call it `LogletCommitResolver`?